Constraint error updating NVD database

running `vulnz database --skipEPSS --skipKEV --skipGHSA` with an already-existing `vulnerabilities.db` on current main (c62debe6bb1fa631efa349383ffba2e1c2034fff). I don't see why it doesn't appear to do an 'update', the code looks OK to me at first glance.

```
HHH000315: Exception executing batch [org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: PROPERTIES.KEY)], SQL: insert into PROPERTIES (VALUE, KEY) values (?, ?)
[SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: PROPERTIES.KEY)
Update Failed
jakarta.persistence.PersistenceException: Converting `org.hibernate.exception.GenericJDBCException` to JPA `PersistenceException` : could not execute batch
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:165)
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:175)
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:182)
	at org.hibernate.internal.SessionImpl.doFlush(SessionImpl.java:1429)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1411)
	at io.github.jeremylong.openvulnerability.VulnerabilityDatabase.commit(VulnerabilityDatabase.java:152)
	at io.github.jeremylong.openvulnerability.VulnerabilityService.loadNVD(VulnerabilityService.java:263)
	at io.github.jeremylong.openvulnerability.VulnerabilityService.updateDatabase(VulnerabilityService.java:102)
	at io.github.jeremylong.vulnz.cli.commands.DatabaseCommand.timedCall(DatabaseCommand.java:214)
	at io.github.jeremylong.vulnz.cli.commands.TimedCommand.call(TimedCommand.java:36)
	at io.github.jeremylong.vulnz.cli.commands.TimedCommand.call(TimedCommand.java:25)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at io.github.jeremylong.vulnz.cli.Application.run(Application.java:73)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:768)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:752)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:314)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1303)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1292)
	at io.github.jeremylong.vulnz.cli.Application.main(Application.java:61)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:65)
Caused by: org.hibernate.exception.GenericJDBCException: could not execute batch
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:61)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:109)
	at org.hibernate.engine.jdbc.batch.internal.BatchingBatch.performExecution(BatchingBatch.java:133)
	at org.hibernate.engine.jdbc.batch.internal.BatchingBatch.doExecuteBatch(BatchingBatch.java:108)
	at org.hibernate.engine.jdbc.batch.internal.AbstractBatchImpl.execute(AbstractBatchImpl.java:153)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.executeBatch(JdbcCoordinatorImpl.java:193)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:641)
	at org.hibernate.engine.spi.ActionQueue.lambda$executeActions$1(ActionQueue.java:483)
	at java.base/java.util.LinkedHashMap.forEach(LinkedHashMap.java:684)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:480)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:329)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:39)
	at org.hibernate.event.service.internal.EventListenerGroupImpl.fireEventOnEachListener(EventListenerGroupImpl.java:107)
	at org.hibernate.internal.SessionImpl.doFlush(SessionImpl.java:1425)
	... 30 common frames omitted
Caused by: org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: PROPERTIES.KEY)
	at org.sqlite.core.DB.newSQLException(DB.java:1030)
	at org.sqlite.core.DB.newSQLException(DB.java:1042)
	at org.sqlite.core.DB.throwex(DB.java:1007)
	at org.sqlite.core.DB.executeBatch(DB.java:832)
	at org.sqlite.core.CorePreparedStatement.executeBatch(CorePreparedStatement.java:64)
	at org.hibernate.engine.jdbc.batch.internal.BatchingBatch.performExecution(BatchingBatch.java:123)
	... 41 common frames omitted
```

(FWIW, the database also has a GHSA property, but that _shouldn't_ be relevant)

--- a/open-vulnerability-store/build.gradle
+++ b/open-vulnerability-store/build.gradle
@@ -15,8 +15,8 @@ dependencies {
     implementation 'com.sun.xml.bind:jaxb-impl:4.0.2'
     implementation 'org.glassfish.jaxb:jaxb-runtime:4.0.2'
     implementation 'com.sun.activation:jakarta.activation:2.0.1'
-    implementation 'org.hibernate.orm:hibernate-core:6.1.7.Final'
-    implementation 'org.hibernate.orm:hibernate-community-dialects:6.1.7.Final'
+    implementation 'org.hibernate.orm:hibernate-core:6.2.7.Final'
+    implementation 'org.hibernate.orm:hibernate-community-dialects:6.2.7.Final'
     implementation 'org.xerial:sqlite-jdbc:3.40.0.0'
 }
 

